app/models/organization.rb

```diff
@@ -120,6 +120,12 @@ def destroyable?
     organization_memberships.count == 1 && articles.count.zero? && credits.count.zero?
   end
 
+  # NOTE: We use Organization and User objects interchangeably. Since the former
```

Comment:
This was part of https://github.com/forem/forem/pull/14079 but I moved it here now.